.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ executors:
         environment:
           JAVA_TOOL_OPTIONS: -Xmx1024m # Nothing to do with surefire plugin, it has its own JVM. The two of these plus ES (1.3 GB) must add up to a bit less than 6GB.
           PGHOST: 127.0.0.1
-      - image: cimg/postgres:13.13
+      - image: cimg/postgres:16.1
```

Comment:
There's another `13:13` on line 272; I wonder if it can be parameterized?

Reply:
I think so, we do that with Java